yast-storage.changes

```diff
@@ -0,0 +1,5 @@
+-------------------------------------------------------------------
+Sat Sep 24 14:47:48 UTC 2016 - dwaas@suse.com
+
+- fix for bsc#910791
```

Comment:
please be a bit more verbose about change you did. Something like `Avoid confusion in small partition warning by displaying also rounded input value (bsc#910791)` and add `- 3.1.104` to next line like other entries.
Then please  make it mergable and I will merge it.